frontend/src/components/Hero/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classnames from 'classnames/bind';
+import styles from './Hero.module.scss';
+
+const cx = classnames.bind(styles);
+
+const Hero = ({ back }) => {
+  return (
+    back == true?
```

Comment:
```suggestion
    back ?
```

이렇게 하면 코드를 줄일 수 있을 것 같아요!